docs: Device ID format warning. (#211)
unravelin/ravelinjs: README.md

});
```

#### Device ID Format

The device ID in the `ravelinDeviceId` cookie or returned by `ravelin.core.id()`
should be treated as an opaque string. Do not attempt to parse or validate the

> format of the ID as we may change it without warning in future.

Reviewer: nit

```suggestion
format of the ID as we may change it without warning in the future.
```

## Upgrading

Note that the format of the deviceId was changed in v1 to include a "rjs-"

> prefix. If you have any validations or parsing that depend on a particular
> format of deviceId, please remove this logic and instead treat the deviceId as
> an opaque string.

Reviewer: nit

```suggestion
prefix. If you do any validation or parsing that checks for a particular
format of the deviceId, please remove this logic and instead treat the deviceId as
an opaque string.
```
